src/main/java/edu/ie3/util/quantities/QuantityUtil.java

```diff
@@ -25,4 +25,51 @@ private QuantityUtil() {
   public static <Q extends Quantity<Q>> ComparableQuantity<Q> makeComparable(Quantity<Q> quantity) {
     return Quantities.getQuantity(quantity.getValue(), quantity.getUnit());
   }
+
+  /**
+   * Compares two {@link Quantity}s, if they are considerably equal. This is foremost important for
+   * {@link tec.uom.se.quantity.DoubleQuantity}s. The comparison is made on the absolute difference
+   * of both quantities' value. Both quantities are converted into a's unit before the comparison.
+   *
+   * @param a First quantity to compare
+   * @param b Second quantity to compare
+   * @param absQuantityTolerance Permissible absolute tolerance
+   * @param <Q> Type of Quantity
+   * @return true, if both quantities' values differ less then the given tolerance else false
+   */
+  public static <Q extends Quantity<Q>> boolean considerablyAbsEqual(
+      Quantity<Q> a, Quantity<Q> b, double absQuantityTolerance) {
+
+    // if units differ, return false
+    if (!a.getUnit().equals(b.getUnit())) return false;
```

Comment:
My initial intention when designing the internal method was to compare two things, that are semantically equal. Therefore, my method didn't have the check for equal units. I will leave it up to you, if you either want to provide an additional method, that is providing this functionality or just remove this line and line 68. BTW: Unit conversion in lines 47 and 71 are pointless, as this line ensures, that the units are already the same.

Reply:
BTW: Added a test case to the test class.